src/components/ApplicationForm/Skills.js

```diff
@@ -145,9 +162,11 @@ export default ({ formInputs, onChange, role, handleResume }) => {
 
       <FormSpacing>
         <QuestionHeading>question 13</QuestionHeading>
-        <SubHeading color="primary">Answer one of the two questions:</SubHeading>
+        <SubHeading color="primary">
+          Answer one of the two questions:<RequiredAsterisk>*</RequiredAsterisk>
```

Comment:
Would be nice if you can pass texts into RequiredAsterisk aka ErrorSpan if possible

Reply:
as props*

Reply:
rESOLVED in my most recent commit, added optional content prop `<Required content='hello' />` but otherwise the content defaults to `*`
<img width="575" alt="Screen Shot 2020-12-14 at 3 35 49 PM" src="https://user-images.githubusercontent.com/38872354/102148872-1d850880-3e22-11eb-993a-3a839225a488.png">